app/models/virtual_cap_pool.rb

```diff
@@ -55,6 +60,11 @@ def add_school_allocation(device_allocation)
     recalculate_caps!
   end
 
+  def remove_school_allocation(device_allocation)
```

Comment:
Naming: `school` is redundant here and maybe we need a bang `!` at the end?

Reply:
The thing is that this removes not all the allocations of the pool but only those of a specific school.